src/interpret/interpret.ml

```diff
@@ -516,7 +512,7 @@ struct
     | Num_type V128 -> V128 V128.zero
     | Ref_type (_null, rt) -> ref_null rt
 
-  (* TODO move to module Env *)
+  (* TODO: remove *)
```

Comment:
Is it still used somewhere?

Reply:
Yes, I still haven't updated `apply` in `exec_extern_func`, I'll need to add the memory id to `Mem`